### ISSUE TITLE
Secret Rooms X-Ray

### ISSUE THREAD

**Issue by WillKracke** [Brycey92/Galaxy-Craft-Issues#48]
**Pack version**
1.0.3

**Describe the bug**
Putting secret rooms trapdoor allows you to see through the world

**To reproduce**
1. Place a wooden secret trapdoor on chiseled granite

**Screenshots and crash reports**
![image](https://user-images.githubusercontent.com/42786612/46920683-03d8a400-cfc0-11e8-8e64-efa2c824965b.png)

**Comment by Brycey92** [comment]
Does this happen when the trap door is next to other materials? Does it happen when other secret rooms blocks are placed next to chiseled granite?

**Comment by WillKracke** [comment]
Yeah I had to leave before I could test, reason for missing info label

**Comment by WillKracke** [comment]
Only occurs with chisel blocks, vanilla blocks work fine.

**Comment by Brycey92** [comment]
Ok, so this happens when any nonsolid Secret Rooms block tries to copy a chisel block. This affects trap doors, ghost blocks, and stairs. It only happens when the nonsolid block is next to the same type of block it copied, and only those real blocks of the same type become xray-ified.

As a stopgap, we can ban these nonsolid blocks, but this should really be fixed at the source.

**Comment by Brycey92** [comment]
I opened an issue [here](https://github.com/Chisel-Team/Chisel/issues/746) on the Chisel github.

**Comment by Brycey92** [comment]
Chisel told me it was a SecretRooms issue: https://github.com/AbrarSyed/SecretRoomsMod-forge/issues/251.
